File: 30DayHackerRankChallengeJava/src/Day20.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class Day20 {

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int[] a = new int[n];
        for (int a_i = 0; a_i < n; a_i++) {
            a[a_i] = in.nextInt();
        }
        // Write Your Code Here
        int count = 0;
        for (int i = 0; i < a.length; i++) {
            for (int j = 0; j < a.length - 1; j++) {
                if (a[j] > a[j + 1]) {
                    int save = a[j];
                    a[j] = a[j + 1];
                    a[j + 1] = save;
                    count++;
                }
            }
        }
        System.out.printf("Array is sorted in %s swaps.\n", count);
        System.out.printf("First Element: %s\n", a[0]);
        System.out.printf("Last Element: %s\n", a[a.length - 1]);
    }
}

File: 30DayHackerRankChallengePython/src/Day20.py
#!/bin/python3

import sys

n = int(input().strip())
a = list(map(int, input().strip().split(' ')))
# Write Your Code Here
numSwaps = 0
for i in range(0, n):
    for j in range(0, n - 1):
        if a[j] > a[j + 1]:
            save = a[j]
            a[j] = a[j + 1]
            a[j + 1] = save
            numSwaps += 1
first = a[0]
last = a[len(a) - 1]
print(f'Array is sorted in {numSwaps} swaps.')
print(f'First Element: {first}')
print(f'Last Element: {last}')
